plutus-pab-client/default.nix

```diff
@@ -1,10 +1,11 @@
 { pkgs, gitignore-nix, haskell, webCommon, webCommonPlutus, buildPursPackage, buildNodeModules, filterNpm }:
 let
+  nativeOnly = pkgs.lib.meta.addMetaAttrs { platforms = with pkgs.lib.platforms; [ linux darwin ]; };
   server-setup-invoker = haskell.packages.plutus-pab.components.exes.plutus-pab-setup;
   server-examples-invoker = haskell.packages.plutus-pab.components.exes.plutus-pab-examples;
   test-generator = haskell.packages.plutus-pab.components.exes.plutus-pab-test-psgenerator;
 
-  generated-purescript = pkgs.runCommand "plutus-pab-purescript" { } ''
+  generated-purescript = pkgs.runCommand "plutus-pab-purescript" { meta.platforms = with pkgs.lib.platforms; [ linux darwin ]; } ''
```

Comment:
nativeOnly?

Reply:
There's a bit of inconsistency with how you handle the `runCommand` instances.

Also, I do think for stuff like this where there's a whole tranche of stuff that won't work in cross, it might be simpler to just not include them when we're cross.